testsuite/MDAnalysisTests/transformations/test_nojump.py

```diff
@@ -0,0 +1,181 @@
+import numpy as np
+import pytest
+from numpy.testing import assert_allclose
+
+import MDAnalysis as mda
+from MDAnalysis.transformations import NoJump, wrap
+from MDAnalysisTests import datafiles as data
+
+
+@pytest.fixture()
+def nojump_universes_fromfile():
+    '''
+    Create the universe objects for the tests.
+    '''
+    u = mda.Universe(data.PSF_TRICLINIC, data.DCD_TRICLINIC)
+    transformation = NoJump()
+    u.trajectory.add_transformations(transformation)
+    return u
+
+
+@pytest.fixture()
+def nojump_universe():
+    """
+    Create the universe objects for the tests.
+    """
+    u = mda.Universe.empty(1, trajectory=True)
+    coordinates = np.empty((100, u.atoms.n_atoms, 3))  # number of frames
+    coordinates[::3, 0] = 0 * np.ones(3) / 3
+    coordinates[1::3, 0] = 1 * np.ones(3) / 3
+    coordinates[2::3, 0] = 2 * np.ones(3) / 3
+    u.load_new(coordinates, order="fac")
+    return u
+
+
+@pytest.fixture()
+def nojump_constantvel_universe():
+    """
+    Create the universe objects for the tests.
+    """
+    Natom = 1
+    Nframe = 100
+    coordinates = np.empty((Nframe, Natom, 3))  # number of frames
+    coordinates[:, 0, 0] = np.linspace(0, 45, Nframe)
+    coordinates[:, 0, 1] = np.linspace(0, 15, Nframe)
+    coordinates[:, 0, 2] = np.linspace(0, 10, Nframe)
+    reference = mda.Universe.empty(Natom, trajectory=True)
+    reference.load_new(coordinates, order="fac")
+    towrap = mda.Universe.empty(Natom, trajectory=True)
+    towrap.load_new(coordinates, order="fac")
```

Comment:
Since I'll be wrapping and unwrapping `towrap`, and need these to be distinct memory locations so I only mutate one set. My understanding of numpy copy semantics is that you aren't actually sure that you create distinct memory locations for a "trajectory".